packages/gasket-utils/lib/index.d.ts

```diff
@@ -153,3 +153,5 @@ export interface ConfigContext {
 
 
 export function warnIfOutdated(pkgName: string, currentVersion: string): MaybeAsync<void>;
+
+export function getPackageLatestVersion(pkgName: string): Promise<string>;
```

Comment:
Maybe also import the type to JSDocs [here](https://github.com/godaddy/gasket/blob/add-missing-type-for-version-check/packages/gasket-utils/lib/get-package-latest-version.js#L3-L8)